src/Form/test-cases/usps_checkout.html

```diff
@@ -1,604 +1,1013 @@
+                        <input name="/atg/store/order/purchase/CouponFormHandler.editCouponErrorURL" type="hidden" value="/store/checkout/shipping.jsp"><input name="_D:/atg/store/order/purchase/CouponFormHandler.editCouponErrorURL" type="hidden" value=" ">
 
+                        <div class="d-flex flex-row justify-content-between total-cost">
 
-                        <option value="WI">WI - Wisconsin</option>
+                            <span class="total-price-label">Total:</span>
 
+                            <span class="total-price-value">
+	        
+	  	  
+		  		  
+	  	  
+			
+			
+			
+		  
+		  
+	        
+             
+                
+                
+                  
+                
+                
+	                
+	                  
+	                
 
 
 
-                        <option value="WY">WY - Wyoming</option>
 
 
 
 
-                        <option value="AA">AA - Armed Forces Americas</option>
 
 
 
 
-                        <option value="AE">AE - Armed Forces Africa</option>
 
 
 
 
-                        <option value="AA">AA - Armed Forces Americas</option>
 
 
+  
 
 
-                        <option value="AE">AE - Armed Forces Canada</option>
 
 
 
 
-                        <option value="AE">AE - Armed Forces Europe</option>
 
 
 
 
-                        <option value="AE">AE - Armed Forces Middle East</option>
+  
+    
+  
+  
 
 
 
 
-                        <option value="AP">AP - Armed Forces Pacific</option>
 
+  
+  
 
 
 
 
+  
+  
+  
+  
+  
+	
 
+  
+  
+    
+  
+  
+  
+  
+  
+    
+    
+  
+    
+    
+      
+      
+      
+      
+      
+        
+        
+        
+        
+        
+        
+        
+        
+        
+          
+            
+          
+          
+        
+        
+      
+      
+      
+        
+        
+          $11.60
+          
+        
+      
 
+     
+    
+  
 
 
 
-                        <script type="text/javascript">
-                            dojo.addOnLoad(function () {
-                                populateState(dojo.byId('atg_store_countryNameSelect'));
-                            });
-                        </script>
+                
+              </span>
+                        </div>
 
+                        <div class="col-12 button-wrapper">
+                            <div class="button-container">
+                                <a id="moveToBillingBtn" href="#" role="button" class="input-btn-primary button--green" data-manual-submit><span>Check Out Now</span></a>
+                                <input tabindex="0" name="/atg/commerce/order/purchase/ShippingGroupFormHandler.createAndMoveToBilling" style="display:none" id="createAndMoveToBilling" type="submit" class="input-btn-primary button--green" value="Check Out Now"><input name="_D:/atg/commerce/order/purchase/ShippingGroupFormHandler.createAndMoveToBilling" type="hidden" value=" ">
```

Comment:
The markup here is strange - the `<a>` is the only thing that's visible, and the `<input type="submit">` is hidden. I'm thinking only the first one should be marked as a submit button?

Reply:
I think in this case both should be marked.